fastlane/Fastfile

```diff
@@ -756,7 +756,8 @@ REPOSITORY_NAME = 'WordPress-Android'
   MAIN_STRINGS_PATH = './WordPress/src/main/res/values/strings.xml'.freeze
   FROZEN_STRINGS_DIR_PATH = './fastlane/resources/values/'.freeze
   LOCAL_LIBRARIES_STRINGS_PATHS = [
-    { library: "Image Editor", strings_path: "./libs/image-editor/ImageEditor/src/main/res/values/strings.xml", exclusions: [] }
+    { library: "Image Editor", strings_path: "./libs/image-editor/ImageEditor/src/main/res/values/strings.xml", exclusions: [] },
+    { library: "WordPress Editor", strings_path: "./libs/editor/WordPressEditor/src/main/res/values/strings.xml", exclusions: [] }
   ].freeze
```

Comment:
Note: I _think_ the fact that we didn't have the lib listed here until now was just an oversight… but would love to have a confidence check and confirmation that this lib was not excluded on purpose (e.g. if it is considered a legacy lib we since replaced with another or whatnot?) since I seem to have a vague recollection about that (though the more I think of it the more I think I might confuse this with editor pods on WPiOS, not libs in WPAndroid… #fridays)

Reply:
> but would love to have a confidence check and confirmation that this lib was not excluded on purpose (e.g. if it is considered a legacy lib we since replaced with another or whatnot?)

Same here, I don't know if that lib might have been excluded on purpose. 🤔

Reply:
Shamelessly also pinging @dcalhoun & @SiobhyB (betrayed by `git blame` as being some of the last devs changing the `strings.xml` file of WordPressEditor 😛 … which makes me think that it's still used—and we still require to include its strings for translations—after all…?)

Reply:
I don't personally recall the file being intentionally left out but, better to reel in to check with @fluiddot since Carlos worked on the translations pipeline recently.

Reply:
Thanks for looping people in 🙂 

Worth nothing that since that last comment, Petros did some more digging and added more details about his discoveries in p5T066-34Q-p2#comment-11838 . Especially it's not super clear to most of us which libs are still used as a local subproject (in `libs/`) vs as a module (published to S3 etc) or even used at all (e.g. legacy and since replaced with something new)…

So I'm interested to have the insights of anyone who might know better which of the libs as projects vs modules are still active vs which should be deprecated etc; as it might be the case that some libs might have been migrated from subproject to module at some point but the subprojects never got properly deprecated?!

(Petros explains all that better that me in the linked P2 comment p5T066-34Q-p2#comment-11838 anyway)

Reply:
As @ParaskP7 mentions, I think it's a good idea to spend some time understanding the dark corners and cleaning up the configuration if needed. 

For this PR, I guess that since we figured out the issue because of untranslated strings, we can safely say that the library is still used and, so, we need to import the strings. 

I run the lane locally and verified that it imports all the strings. I think there are a few strings that we don't need in WordPress, so we could set the attribute that prevents them from being imported if we want to keep the client's `strings.xml ` clean, but I didn't find any duplicates, so it shouldn't cause any issue anyway.

Reply:
> Shamelessly also pinging @SiobhyB (betrayed by git blame as being some of the last devs changing the strings.xml file of WordPressEditor 😛 … which makes me think that it's still used—and we still require to include its strings for translations—after all…? 

Apologies for arriving late! 

For (possibly-not-useful-at-this-stage) context, my understanding is that any strings added to files under [the WordPress Editor library](https://github.com/wordpress-mobile/WordPress-Android/tree/trunk/libs/editor/WordPressEditor), such as [GutenbergEditorFragment.java](https://github.com/wordpress-mobile/WordPress-Android/blob/558badc2229419091a255857fbed5b1138b27324/libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergEditorFragment.java), would need to be added to the library's strings.xml file. The addition of strings here would be infrequent (as it is rare we need to add strings on the native side vs. Gutenberg), which may be why it appears that the file isn't in use. It does serve a purpose, though, most notably for dialogs that are implemented on the native side.

I'm afraid I don't have any insight into the questions around which files are now deprecated.

Reply:
> I don't personally recall the file being intentionally left out but, better to reel in to check with @fluiddot since Carlos worked on the translations pipeline recently.

I'm afraid I can't provide further insights about this issue, as the recent translation pipeline work was tightly focused on fixing translation issues from the JS/Gutenberg side. However, I recall we identified some missing strings as pointed in https://github.com/wordpress-mobile/WordPress-Android/issues/15822 but we didn't investigate them deeper. 

Something I noticed back then was that some strings were duplicated in both [libs/editor/WordPressEditor/src/main/res/values/strings.xml](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/libs/editor/WordPressEditor/src/main/res/values/strings.xml) and [src/main/res/values/strings.xml](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/res/values/strings.xml) files, which didn't help to uncover this issue sooner as those strings are already translated. It would be also interesting to review the strings located in [libs/editor/WordPressEditor/src/main/res/values/strings.xml](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/libs/editor/WordPressEditor/src/main/res/values/strings.xml) and identify which ones are really in use, as probably some of them are no longer required.

Reply:
> For (possibly-not-useful-at-this-stage) context, my understanding is that any strings added to files under [the WordPress Editor library](https://github.com/wordpress-mobile/WordPress-Android/tree/trunk/libs/editor/WordPressEditor), such as [GutenbergEditorFragment.java](https://github.com/wordpress-mobile/WordPress-Android/blob/558badc2229419091a255857fbed5b1138b27324/libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergEditorFragment.java), would need to be added to the library's strings.xml file. 

This is actually very useful as it suggests me that… the role of the WordPressEditor lib is to somewhat be a bridge between Gutenberg and the native side?

> The addition of strings here would be infrequent (as it is rare we need to add strings on the native side vs. Gutenberg), which may be why it appears that the file isn't in use. It does serve a purpose, though, most notably for dialogs that are implemented on the native side. (Although, it's possible I am mistaken in my understanding and there's somewhere else the strings could be added.)

That explains why we hadn't noticed this lib's `strings.xml` being missed by our tooling when it merges the strings to be sent to translation, as most of those strings had probably been translated a while ago, and only the new strings were thus not sent to translation anymore, but since those are infrequent, that's likely one of the main reason we didn't spot them missing earlier that this sprint.

> Something I noticed back then was that some strings were duplicated in both [libs/editor/WordPressEditor/src/main/res/values/strings.xml](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/libs/editor/WordPressEditor/src/main/res/values/strings.xml) and [src/main/res/values/strings.xml](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/res/values/strings.xml) files, which didn't help to uncover this issue sooner as those strings are already translated

That's interesting and something I'll keep in mind when doing the code freeze later today (I was planning to check the possilibity of some of WPE's lib strings overwriting the app's ones after this change anyway, as Lore also suggested above)

> It would be also interesting to review the strings located in [libs/editor/WordPressEditor/src/main/res/values/strings.xml](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/libs/editor/WordPressEditor/src/main/res/values/strings.xml) and identify which ones are really in use, as probably some of them are no longer required.

Yes please, we definitively need to do a deeper investigation to get rid of useless strings (especially if some of them are duplicates from ones from the main app), but also to clarify and tidy all that to make it clear and explicit what libs, strings and code is now unused and deprecated and can be deleted vs what is the real source of truth.

Reply:
> ...the role of the WordPressEditor lib is to somewhat be a bridge between Gutenberg and the native side? (especially with alerts and stuff)

Ah @AliSoftware, I can provide context on that. So, the WordPressEditor lib is meant to be a "host" library for the editor(s) in the WP app. It has been traditionaly be harbouring at least 2 editor at the time (it was **4** at some point in the past). As I write this, is is hosting code and interfaces for 2 editors, the "classic" Aztec based editor, and the current default (the Gutenberg block editor).

In terms of "native side" or not, the WordPressEditor lib is the part of the native side we think is closer to the WP app side, rather than the native that is there [to support gutenberg-mobile](https://github.com/WordPress/gutenberg/tree/trunk/packages/react-native-bridge/android). In other words, gutenberg-mobile also has native parts, just closer to the block editor per se.

In a kinda schematic way:
<table>
<tr><th colspan="3">Native side</th><th colspan="2">JS side</th></tr>
<tr><td>WP App</td><td>WordPressEditor lib</td><td>Gutenberg bridge</td><td>JS side of the bridge</td><td>Gutenberg RN editor</td></tr>
</table>

Reply:
> This is actually very useful as it suggests me that… the role of the WordPressEditor lib is to somewhat be a bridge between Gutenberg and the native side? (especially with alerts and stuff)

That's right! Most of the dialogs implemented here are also media-related, and I think media-related requests are a good example of when it's necessary to lean on the native side for function calls and data, but still update the Gutenberg side.